pkgs/by-name/ce/certdump/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildDotnetModule,
+  fetchFromGitHub,
+  stdenv,
+}:
+
+buildDotnetModule rec {
+  pname = "certdump";
+  version = "unstable-2023-12-25";
```

```suggestion
  version = "0-unstable-2023-12-25";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning